std_msgs/Header header
  uint32 seq
  time stamp
  string frame_id
geometry_msgs/Point FL_foot
geometry_msgs/Point FR_foot
geometry_msgs/Point RL_foot
geometry_msgs/Point RR_foot
